view/src/extras/dealStepper.js

```diff
@@ -54,16 +54,13 @@ export default function HorizontalLinearStepper(props) {
 
   /** Sets the last step (so formatting can change on last step) */
   const isLastStep = (step) => {
-    return step === 2;
+    return step === steps.length - 1;
   };
 
   /** Moves to the next step (page) in the Stepper */
   const handleNext = () => {
-    // Confirm that selected (needs to be connected to child)
-    if (false) {
-      props.alert("warning", "Please select a Surplus Object to continue.");
-      return;
-    }
+    // ADD AN IF STATEMENT TO ENSURE THAT SURPLUS CARD IS SELECTED BEFORE
+    // MOVING ON
```

Comment:
similar to whatever other PR i just reviewed, let's use `TODO(name): Do the thing` syntax.

I don't like when my code shouts at me :)